examples/assignment-1/src/main/java/ro/utcn/spet/example/a1/entity/Userss.java

```diff
@@ -0,0 +1,27 @@
+package ro.utcn.spet.example.a1.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Data
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name ="userss")
+public class Userss {
```

Comment:
Why the double ss at the end? Also, usually classes are at singular (not plural). The only case in which you would have plural names is in case of collection wrappers.